tests/WE2E/machine_suites/comprehensive.derecho

```diff
@@ -48,6 +48,7 @@ grid_RRFS_CONUScompact_13km_ics_HRRR_lbcs_RAP_suite_RRFS_v1beta
 grid_RRFS_CONUScompact_25km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v16
 grid_RRFS_CONUScompact_25km_ics_HRRR_lbcs_HRRR_suite_HRRR
 grid_RRFS_CONUScompact_25km_ics_HRRR_lbcs_RAP_suite_RRFS_v1beta
+grid_RRFS_CONUScompact_25km_ics_RRFS_lbcs_RRFS_suite_RRFS_v1beta
```

Comment:
I still do not think that this expensive ensemble test should be run on every platform via the comprehensive test suite. Please pick on machine.

Reply:
Hi @christinaholtNOAA, WE2E tests need to be added to all comprehensive WE2E test suites.  The comprehensive test suite is all tests that can be run on a given system.  The previous issue was that @natalie-perlin had added the new WE2E test to all coverage WE2E test suites.  Having the new test in all comprehensive test suites and only the `coverage.orion` suite is the correct setup.